sympy/sets/fancysets.py

```diff
@@ -483,6 +492,127 @@ def _intersect(self, other):
             else:
                 return
 
+    def _union(self, other):
+        """
+
+        This function should only be used internally.
+        See Set._union for docstring
+
+        Used within the :class:`Union` class
+
+        Returns
+        =======
+        self._union(other) returns a new, joined ImageSet if self knows how
+        to join itself with other, otherwise it returns ``None``.
+
+        Examples
```

Comment:
These examples are already present in the docstrings of `ImageSet`.
Hence, they are redundant.
Moreover, is adding docstring to internal functions a good practice ?
Ping @hargup @aktech